grype/presenter/models/vulnerability_metadata.go

```diff
@@ -26,14 +44,41 @@ func NewVulnerabilityMetadata(id, namespace string, metadata *vulnerability.Meta
 	}
 
 	return VulnerabilityMetadata{
-		ID:          id,
-		DataSource:  metadata.DataSource,
-		Namespace:   metadata.Namespace,
-		Severity:    metadata.Severity,
-		URLs:        urls,
-		Description: metadata.Description,
-		Cvss:        NewCVSS(metadata),
+		ID:             id,
+		DataSource:     metadata.DataSource,
+		Namespace:      metadata.Namespace,
+		Severity:       metadata.Severity,
+		URLs:           urls,
+		Description:    metadata.Description,
+		Cvss:           NewCVSS(metadata),
+		KnownExploited: toKnownExploited(metadata.KnownExploited),
+	}
+}
+
+func toKnownExploited(knownExploited []vulnerability.KnownExploited) []KnownExploited {
+	result := make([]KnownExploited, len(knownExploited))
+	for idx, ke := range knownExploited {
+		result[idx] = KnownExploited{
+			CVE:                        ke.CVE,
+			VendorProject:              ke.VendorProject,
+			Product:                    ke.Product,
+			DateAdded:                  formatToDate(ke.DateAdded),
+			RequiredAction:             ke.RequiredAction,
+			DueDate:                    formatToDate(ke.DueDate),
+			KnownRansomwareCampaignUse: ke.KnownRansomwareCampaignUse,
+			Notes:                      ke.Notes,
+			URLs:                       ke.URLs,
+			CWEs:                       ke.CWEs,
+		}
+	}
+	return result
+}
+
+func formatToDate(t *time.Time) string {
+	if t == nil {
+		return ""
 	}
+	return t.Format("2006-01-02")
```

Comment:
Does it make sense to use `time.DateOnly` here? https://pkg.go.dev/time#pkg-constants

I had to look up to figure out why this particular string literal is involved :)